### ISSUE TITLE
isnet:imagelevel.py 

### ISSUE THREAD

**Issue by shujunyy123** [SegmentationBLWX/sssegmentation#15]
imagelevel.py :
47: feats_il = self.correlate_net(x, torch.cat([x_global, x], dim=1))

isanet.py：
47：context = super(SelfAttentionBlock, self).forward(x, x)

is there any problem？  bug？
